content/id/docs/Catalyst/kegiatan-eastern-townhall-indonesia.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Kegiatan Eastern Townhall Indonesia"
+linkTitle: "Kegiatan Eastern Townhall Indonesia"
+hide_feedback: true
+weight: 4
+description:
```

Comment:
comment out or remove the description values in the front matter for each of the mardown files you added if your not going to use it.